addons/project/views/project_task_views.xml

```diff
@@ -31,7 +31,6 @@
                         <filter name="create_date_last_365_days" string="Last 365 Days" domain="[('date_last_stage_update', '&gt;', datetime.datetime.combine(context_today() - relativedelta(days=365), datetime.time(23, 59, 59)).to_utc())]"/>
                     </filter>
                     <separator/>
-                    <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
```

Comment:
Hello @deneuvillem

Yes I agree that it’s a good approach to keep the filter in the base search view and simply hide it in the project sharing and reporting views. However rather than just hiding it I think it would be more efficient to remove the filter from the project sharing and project/FSM report views entirely to avoid unnecessary complexity. With this approach I’ve adjusted the code accordingly removing the filter from the project sharing and project/FSM report views.

I hope I’ve addressed all your doubts and I’d be happy to hear any further suggestions you might have!

Thank you 😄

Reply:
Well we can go for your solution then :smile: As you work in master, I guess it's okay to move filters from a view to another